cbi/oracle/oracle_cbi/oracle_cbi.pt

```diff
@@ -0,0 +1,273 @@
+name "Oracle Cloud Common Bill Ingestion"
+rs_pt_ver 20180301
+type "policy"
+short_description "Downloads cost reports from Oracle Cloud (OCI) and then uploads them to a Flexera CBI endpoint. See the [README](https://github.com/flexera-public/policy_templates/tree/master/cbi/oracle/oracle_cbi) and [docs.flexera.com/flexera/EN/Automation](https://docs.flexera.com/flexera/EN/Automation/AutomationGS.htm) to learn more."
+severity "low"
+category "CBI"
```

Comment:
Not sure where the options for category are defined, but agree this doesn't really fit under any of the existing categories. Could be `cost` maybe? 

Thoughts @bryankaraffa @taterdog590 ?

Reply:
@nia-vf1   my suggestion is to use Cost for the Category